src/benchmarklib/benchmark_runner.cpp

```diff
@@ -61,10 +63,15 @@ void BenchmarkRunner::run() {
   _table_generator->generate_and_store();
 
   if (_config.verify) {
+    std::cout << "- Loading tables into SQLite for verification." << std::endl;
     // Load the data into SQLite
     _sqlite_wrapper = std::make_unique<SQLiteWrapper>();
     for (const auto& [table_name, table] : StorageManager::get().tables()) {
+      std::cout << "- Loading '" << table_name << "' into SQLite " << std::flush;
+      Timer timer;
       _sqlite_wrapper->create_table(*table, table_name);
+      std::cout << "(" << format_duration(std::chrono::duration_cast<std::chrono::nanoseconds>(timer.lap())) << ")"
```

Comment:
With this pattern being used over and over again, shouldn't the timer have a `lap_as_string` method? Name tbd?

Reply:
I like `lap_formatted()`, but whatever you prefer

Reply:
Sounds good